### DISCUSSION TITLE
throttle middleware supports seconds as decay

### DISCUSSION THREAD

**Discussion by jjsty1e** [laravel/framework#33931] [General]
for some situations, we should limit ours users to not make duplicated request at the same time , for example , if a user make 3 requests in 1 seconds (to the same route), we just want to accept the first and drop the others.

but now using the throttle middleware, we have the minus time is 1 minute(60 seconds), I hope it can be more accurate(using seconds or millinseconds the better)

at the first I'm thinking to create a `DebounceRequests` middleware to do this, but if throttle middleware support seconds as decay, it can be implemented originaly.

**Comment by guilhermebillig** [comment]
Now it is possible to use float for seconds, like 0.5 = 30 seconds.
